hack/ovn-kind-cno.sh

```diff
@@ -90,7 +99,7 @@ if [ "$BUILD_OVN" = true ]; then
   pushd dist/images
   sudo cp -f ../../go-controller/_output/go/bin/* .
   cat << EOF | docker build -t origin-ovn-kubernetes:dev -f - .
-FROM quay.io/openshift/origin-ovn-kubernetes:4.5
+FROM quay.io/openshift/origin-ovn-kubernetes:4.6
```

Comment:
`quay.io/openshift/origin-ovn-kubernetes:latest` vs. `4.6`?

Reply:
I do not recommend to use latest and better pin the versions, so you always know what you are using and everybody has the same version

Reply:
I don't know which is right, but @rcarrillocruz push #813 to move them all to latest.

Reply:
Using latest is fine. These scripts are used mainly for development, i.e. master, so it's more than likely that we want to test CNO latest with everything latest.
If we set to 4.6 then it's a moving target, every release we need to change it. Even within the same release, there may be times where a breaking change doesn't work against CNO master.

Reply:
fair enough,

Reply:
I'd rather keep versions pinned correctly in appropriate branches:
https://github.com/openshift/cluster-network-operator/pull/814#issuecomment-701551960

Reply:
Ok, yeah that makes sense. Let's pin the versions for each branch appropriately. it just becomes a maintenance issue over time. does it make sense to create a version file for each release branch and then pick the version as a tag in each of the clients?